### ISSUE TITLE
Players losing homes!

### ISSUE THREAD

**Issue by goncalomano** [brunyman/Essentials-MySQL-Storage-Extension#7]
I installed the plugin , all fine but 2 weeks later everyone started losing their homes , is there a way to disable home saving to MYSQL? Because there is no real use for a cross-server home sync . Thanks

**Comment by CharlyZest** [comment]
I have the same problem, I have noticed that the DB file and registry are fine until this legend enters and leaves [Essentials] Found new UUID for XXXXXXX. Replacing d07e177a-457d-3a37-9b45-4f759e5b6b7e with 42ba52c9-edd6-31ca-a40b-d952f3f4c5e5.

Awaiting your comments, greetings.

**Comment by iLuckyx** [comment]
Having the same issue. Any way we can prevent this from happening?

**Comment by Danielpeter-99** [comment]
I can support this issue. Same thing happening to me.

**Comment by Louploup31** [comment]
I have the same problem, is it possible to have a solution please, knowing that the plugin is not free?

**Comment by itsmemac** [comment]
same issue.. :( all my players lost their home

**Comment by Louploup31** [comment]
I deleted the plugin knowing that I have no answer and I let the essentials register the homes of each server and I have no more problems

**Comment by Danielpeter-99** [comment]
I have done the same thing.

**Comment by iLuckyx** [comment]
> I deleted the plugin knowing that I have no answer and I let the essentials register the homes of each server and I have no more problems

I agree. I removed this plugin and the issue has not occurred since. And that was about 2 months ago at this point.

**Comment by AJ-Ianozi** [comment]
This has also been reported on my server, I was digging to try and find out why it happened, narrowed it down to this.  I just updated to the latest one, any updates brunyman?

**Comment by brunyman** [comment]
I need more data about your server setup, server and Essentials version EMS config to try to reproduce it and debug it

**Comment by iLuckyx** [comment]
That's the issue, we.. or at least I don't know how to reproduce on demand. I spent hours trying to provide whatever information I could to help debug this but it just seems to be random after using it for a while. I currently don't use this due to many many times of players losing homes. Since this plugin was removed back in Feb. The issue of homes being yeeted out of no where has not happened again. As mentioned above, it seems when UUID are being updated but even that I couldn't reproduce myself. I know this isn't much help, but just wanted to give some more context. Hope we can get this fixed as this plugin was a life savior when it was being used. ❤️ Thanks for your hard work, duders.

**Comment by brunyman** [comment]
Question is, why would players UUID's change in the first place? This should not happen, if it does it will be treated as a new player as data in the database is stored based on their UUID's, also for the sync to work across other server they need to have the same UUID on all servers.

**Comment by brunyman** [comment]
> I have the same problem, I have noticed that the DB file and registry are fine until this legend enters and leaves [Essentials] Found new UUID for XXXXXXX. Replacing d07e177a-457d-3a37-9b45-4f759e5b6b7e with 42ba52c9-edd6-31ca-a40b-d952f3f4c5e5.
> 
> Awaiting your comments, greetings.

Player's UUID's should not change, read more info here, you may need to proper configure firewall and only allow connections to your server from your bungee only:
https://www.spigotmc.org/threads/uuid-spoofing.66277/

If UUID's do change players will lose data as data is stored based on their UUID.
